### ISSUE TITLE
Sheila making strange noise and not cutting properly

### ISSUE THREAD

**Issue by JackiePease** [DoESLiverpool/somebody-should#1715]
![IMG_20220709_112937](https://user-images.githubusercontent.com/12802256/178102138-dbcce705-f2bc-4f4e-8d25-30942f16a42d.jpg)
Sheila engraving ok, but makes strange noise when cutting as if catching on something.

**Comment by JackiePease** [comment]
@DoESLiverpool/laser-maintenance

**Comment by JackiePease** [comment]
I spoke to Klara who has been using this today.

She says this issue has been occurring intermittently for a while and can be temporarily fixed by power cycling.

This is still an issue, but I'll take the broken panel off.

The lid switch error has reoccurred. @huffeec has fixed temporarily by straightening the hinge?

**Comment by FatihDeveci** [comment]
Lid switch error still persists.

The issue "Making strange noise when cutting as if catching on something" also still persists. It is very similar to the Gerald issues in late 2020 (see #1503 ). Laser head jumps/skips after the noise and whole drawing shifts.

To get rid of this issue, after long attempts, I sometimes combined small lines to make a polyline (path), and sometimes split polylines into lines. In this way, the issues were not solved but decreased during my session.

**Comment by magman2112** [comment]
I did find that one of the drive belts (y-axis, or side to side) had a low tension, which could cause it to jump on the drive wheel on fast movements. I have now re-tensioned this belt. This should hopefully fix the skipping issue.

@Sean-anotherone also adjusted the magnet on the lid, there still appears to be an issue with the lid though, so we are looking to replace the switch, once we have sourced a suitable replacement.
